liota/dcc/vrops.py

```diff
@@ -155,26 +156,25 @@ def publish(self, metric):
         self.con.send(message)
 
     def init_relations(self, gw):
-      """ This function initializes all relations between gateway and it's children.
-          It is called after each object's UUID is received.
+        """ This function initializes all relations between gateway and it's children.
```

Comment:
I am not sure if the spaces are added intentionally. Please use a python code editor to format the code.